tests/rules/parsing_method.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file Test for no-unsanitized rule
+ * @author Frederik Braun et al.
+ * @copyright 2015-2017 Mozilla Corporation. All rights reserved
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../lib/rules/parsing_method");
+const RuleTester = require("eslint").RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const eslintTester = new RuleTester();
+
+eslintTester.run("parsing_method", rule, {
+    // Examples of code that should not trigger the rule
+    // XXX this does not find z['innerHTML'] and the like.
+
+    valid: [
+        {
+            code: "var doc = parser.parseFromString('static string');",
+        },
+        {
+            code: "var doc = Document.parseHTMLUnsafe('static string');",
+        },
+    ],
```

Comment:
What would a sanitized version of this look like?

`parseHTMLUnsafe(sanitize(badness))` or would `sanitizeDocument(parseHTMLUnsafe(badness))` be more likely? I am leaning to (1) and would like to suggest to add a simplified test case here